packages/auto-approve/src/interfaces.ts

```diff
@@ -54,6 +55,16 @@ export interface Versions {
   newMinorVersion: string;
 }
 
+/**
+ * Interface for return type of array from getTargetFiles. It is the amalgamation
+ * of the file that was changed in the PR, plus the file rule that applies
+ * to that given file.
+ */
+export interface fileAndMetadata {
```

Comment:
```suggestion
export interface FileAndMetadata {
```

Let's keep interfaces/classes capitalized